charts/cluster-api-provider-aws/values.yaml

```diff
@@ -1,6 +1,8 @@
-aws:
- encodedCredentials: ""
- iamRole: ""
+awsEncodedCredentials: ""
+awsIAMRole: ""
+ocmToken: ""
 manager:
+  cmd: /bin/cluster-api-provider-aws-controller-manager
```

Comment:
I have no problem with it, but:
 * Why introduce new variables?
 * Where is the boundary for deciding when to use a variable and when not to?

Reply:
I added the command to be able use the upstream CAPA image , the binary in CAPA image registry.k8s.io/cluster-api-aws/cluster-api-aws-controller is ./manager